api/envoy/api/v2/route/route.proto

```diff
@@ -1038,6 +1041,13 @@ message DirectResponseAction {
   core.DataSource body = 2;
 }
 
+message Noop {
+      // If set to true, then when this route is hit, stream info (and hence access logs)
+      // will be populated with the name of this route. Only the NOOP routes that set this
+      // flag will be added to access logs.
```

Comment:
Where in stream info is this added? Stream info is also largely an internal construct, the docs here should largely be reflecting of what's going on from an operator perspective, e.g. does this appear in access logs? How does it appear in access logs? What's the magic escape sequence to see it for text access logs, etc? I would drop reference to stream info.